tests/data/pattern_matching_extras.py

```diff
@@ -77,3 +77,8 @@ def func(match: case, case: match) -> case:
 match match:
     case case:
         pass
+
+
+match a, *b(), c:
```

Comment:
perhaps also add a test case with `*a`, and no b?

Reply:
That is not a valid syntax.
```py
>>> match *a:
  File "<stdin>", line 1
    match *a:
            ^
SyntaxError: invalid syntax
```
For one regular starred and one normal mixed, see the same file. We have an example for that.

Reply:
Ah my bad, I should perhaps play with pattern matching for once haha.

Reply:
Well then, I need to play with pattern matching too.